idd/Energy+.idd.in

```diff
@@ -20899,14 +20955,17 @@ People,
        \note optional (fifth thermal comfort model and report type)
 
 ComfortViewFactorAngles,
-       \memo Used to specify radiant view factors for thermal comfort calculations.
+       \memo Used to specify radiant view factors for thermal comfort calculations. Up to 20 AngleFactors are accepted.
+       \extensible:2
+       \max-fields 42
```

Comment:
@jmarrec This can be made fully extensible by removing the limit of 20 surfaces in [`GetAngleFactorList`](https://github.com/NREL/EnergyPlus/blob/b4443c679af7f30be975a7c5619ac964b39474f2/src/EnergyPlus/ThermalComfort.cc#L1691). See #8589 .